src/crimson/osd/osd_operations/peering_event.h

```diff
@@ -65,7 +65,7 @@ class PeeringEvent : public OperationT<PeeringEvent> {
 
 public:
   template <typename... Args>
-  PeeringEvent(
```

Comment:
why? `explicit` makes difference only if there is a single parameter for the ctor. And I cannot find the explanations in the commit message for this change.

Reply:
when build it always find the fist definition, can't find the constructor with delay parameter.  has build error: can't find the matched definition.

Reply:
your are right, seems last time build I made some mistake, it only can find the first ctor, I try again. explicit no useful here. remove them.